### PR TITLE
fix(docs-infra): remove polyfill web-animations since we use Angular …

### PR DESCRIPTION
**Pull request by wKoza**
…> 6 and we don't use `AnimationBuilder`

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [X] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Currently, we import the polyfill 'web-animation' for the older browsers (not es2015); Since Angular 6, this polyfill doesn't have to be imported.

## What is the new behavior?
The polyfill has been removed;

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: aio/src/ie-polyfills.js [hunk 1]
```diff
@@ -16,6 +16,3 @@ import 'core-js/es6/set';
 
 /** IE10 and IE11 requires the following for NgClass support on SVG elements */
 import 'classlist.js';
-
-/** IE10 and IE11 requires the following to support `@angular/animation`. */
-import 'web-animations-js';
```
Comment (gkalpak) [inline aio/src/ie-polyfills.js]: So, `web-animations-js` is only needed for `AnimationBuilder`? Is that documented somewhere?
Wouldn't it be better to comment this out instead, so it is easier to know what to do if we need it in the future?

Comment (wKoza) [reply aio/src/ie-polyfills.js]: yes [here](https://angular.io/guide/browser-support#optional-browser-features-to-polyfill) :clap:  !
> For example, the animations library relies on the standard web animation API, which is only available in Chrome and Firefox today. (note that the dependency of web-animations-js in Angular is only necessary if AnimationBuilder is used.)

You can see this [commit ](https://github.com/angular/angular/commit/b2f366b3b795d2e43fb3b54c13554f95779f8ee4).

Sure, I can comment this line and add an explain.

**Review by gkalpak — APPROVED** [review]
The fixup commit has the wrong message(`fixup! fix(docs-infra): token window`) and won't be squashed into the first commit.

**Comment by wKoza** [comment]
@gkalpak, if I don't grab the good number ... done

**Review by matsko — APPROVED** [review]

**Pull request closed without merging**